costar_dataset/block_stacking_reader_torch.py

```diff
@@ -885,6 +979,19 @@ def ConvertImageListToNumpy(data, format='numpy', data_format='NHWC'):
                 raise ValueError('Unsupported input data for y: ' + str(x))
 
             # Assemble the data batch
+            # HACK(rexxarchl): Squeeze to match the output from tensorflow
```

Comment:
how do the outputs differ? The answer to that question might be useful to specify in the docstring, & include a link to the relevant pytorch/tf docs